rope/contrib/autoimport/sqlite.py

```diff
@@ -113,25 +133,28 @@ def __init__(
                 autoimport = AutoImport(..., memory=True)
         """
         self.project = project
+        self.prefs = self.project.prefs.autoimport
         project_package = get_package_tuple(project.root.pathlib, project)
         assert project_package is not None
         assert project_package.path is not None
+        if underlined is not None:
+            self.prefs.underlined = underlined
```

Comment:
Don't we need to copy the `self.prefs` here if we're going to mutate it? Otherwise it would change the setting globally and in other instances of `AutoImport` that shares the same prefs.

Reply:
Good point - I ran a deep copy. Maybe in the future, it should automatically reload on pyproject.toml changes? Would you want me to include that in this PR?